internal/core/command/device.go

```diff
@@ -16,28 +16,41 @@ package command
 
 import (
 	"context"
+	"fmt"
 	"net/http"
 
+	"github.com/edgexfoundry/edgex-go/internal"
+	"github.com/edgexfoundry/edgex-go/internal/core/command/config"
+	"github.com/edgexfoundry/edgex-go/internal/core/command/errors"
+	"github.com/edgexfoundry/edgex-go/internal/core/command/interfaces"
+
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
 	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
 
-	"github.com/edgexfoundry/edgex-go/internal/core/command/config"
-	"github.com/edgexfoundry/edgex-go/internal/core/command/errors"
-	"github.com/edgexfoundry/edgex-go/internal/core/command/interfaces"
+	"github.com/gorilla/mux"
 )
 
 func executeCommandByDeviceID(
-	ctx context.Context,
-	deviceID string,
-	commandID string,
+	r *http.Request,
 	body string,
-	queryParams string,
-	isPutCommand bool,
 	lc logger.LoggingClient,
 	dbClient interfaces.DBClient,
-	deviceClient metadata.DeviceClient) (string, error) {
+	deviceClient metadata.DeviceClient,
+	httpCaller internal.HttpCaller) (string, error) {
+
+	var ctx context.Context
+
+	if r == nil {
+		return "", errors.NewErrExtractingInfoFromRequest()
+	}
+
+	ctx = r.Context()
```

Comment:
Can we use `:=` here and eliminate the declaration(`var`) above to simplify the code and make it consistent with the rest of the code base?